### PR TITLE
added pip upgrade instructions

### PR DESCRIPTION
**Pull request by babbush**
Fixes https://github.com/quantumlib/OpenFermion/issues/38

Also fixed two pep8 violations in the measurement module.

### CODE REVIEW

**Comment by babbush** [comment]
Actually I changed my mind, we already say to make sure one is using the latest version of pip.

**Comment by babbush** [comment]
Decided that we will write this in the intro docs but not readme.

File: README.rst [hunk 1]
```diff
@@ -19,8 +19,9 @@ OpenFermion is an open source effort for compiling and analyzing quantum algorit
 Getting started
 ---------------
 
-Installing OpenFermion requires pip. Make sure that you are using an up-to-date version of it.
-To install the latest development version of OpenFermion, clone `this <http://github.com/quantumlib/OpenFermion>`__ git repo,
+Installing OpenFermion requires pip. Make sure that you are using an up-to-date
+version of it. To install the latest development version of OpenFermion,
+clone `this <http://github.com/quantumlib/OpenFermion>`__ git repo,
```
Comment (DanielSank) [inline README.rst]: Doesn't look like anything actually changed here.

Comment (babbush) [reply README.rst]: I see, it appears that a review is required from somebody with merge rights. I wish I could set those things separately but I guess not.

**Pull request merged**